cvat-core/src/api-implementation.ts

```diff
@@ -183,6 +183,8 @@ export default function implementAPI(cvat) {
             sort: isString,
             search: isString,
             jobID: isInteger,
+            task_id: isInteger,
```

Comment:
probably that should be taskID field. 
I could not find it is used anywhere as ``task_id`` before, so, we may try to rename it